app/build.gradle

```diff
@@ -190,9 +190,9 @@ dependencies {
     compile "com.google.firebase:firebase-database:${firebaseVersion}"
     compile "com.google.android.gms:play-services-auth:${firebaseVersion}"
 
-    def androidArch = '1.0.0-alpha7'
```

Comment:
The name was androidArch because the version involves all components, like supportLibraryVersion, but no problem with roomVersion now. If we add more arch things, I'll suggest to change.